client/components/UI/Form/ToggleInput/style.scss

```diff
@@ -1,6 +1,21 @@
+@import '~superdesk-core/styles/sass/mixins.scss';
+@import '~superdesk-core/styles/sass/variables.scss';
+
 .sd-line-input__toggle {
     .sd-line-input__label {
         margin-top: 4px;
         max-width: none;
     }
 }
+
+.sd-toggle--withFocus {
```

Comment:
Should these SCSS classes be under `components/UI/Toggle/style.scss` instead of the Input field?